server/data/activeLduClient.js

```diff
@@ -0,0 +1,23 @@
+const db = require('./dataAccess/db')
+
+module.exports = {
+  async addLdu(lduCode) {
+    const query = {
+      text: `INSERT INTO active_local_delivery_units (ldu_code) VALUES ($1) ON CONFLICT (ldu_code) DO UPDATE SET ldu_code=active_local_delivery_units.ldu_code RETURNING ldu_code`,
+      values: [lduCode],
+    }
+    const response = await db.query(query)
+    return response
+  },
+
+  async doesLduExist(lduCode) {
+    const query = {
+      text: `SELECT ldu_code FROM active_local_delivery_units WHERE ldu_code=$1`,
+      values: [lduCode],
```

Comment:
In use of force we do something like this: 
```const getNumberOfPendingStatements = async (reportId, client = nonTransactionalClient) => {
  const { rows } = await client.query({
    text: `select count(*) from statement where report_id = $1 AND statement_status = $2`,
    values: [reportId, StatementStatus.PENDING.value],
  })
  return parseInt(rows[0].count, 10)
}```
by using count we can always assume that a row will exist.

Reply:
There is a general convention to name methods which return boolean `is*` - so in this case could call it something like `isLduPresent`

Reply:
changed to isLduPresent

Reply:
I'm not 100% sure about the other point - using count, because it attempts to access [0] when there may be no rows at all. 
I'm thinking this might be better
    **const {rows} = await db.query(query)
    return getIn(rows, [[0]]) ? 'true' : 'false'**

Reply:
done

Reply:
done